Src/BootCamp.Chapter/Calculator/Bmi.cs

```diff
@@ -0,0 +1,26 @@
+﻿namespace BootCamp.Chapter.Calculator
+{
+    public class BmiCalculator
```

Comment:
This is not a calculator, this is a calculation to. In any case, it's wrong.
It's bad, because for bmi, you do not need a whole person. Weight and height is all you need.
Bmi itself should be something that is returned and not stored in an object.